src/renderer/components/settings/field/password.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+    {{ field.display || field.name }}
+    <button @click="editing = !editing"></button>
+    <div v-if="editing">
+        <label>
+            <input
+                type="password"
+                :name="field.name"
+                :minLength="field.minLength"
+                :maxLength="field.maxLength"
+                :value="value"
+            />
+        </label>
+        <button @click="$emit('value-change', sha256(value))"></button>

```

Comment:
Usually it's is best practice to hash password with a hash. such that you generate a random string of lets say 5-10 alpha numeric chars called salt and then run the hash function on `password + salt`. This prevents from a attacker that put his hands on the hashes to crack all the passwords together. This is critical for bigger collections of passwords and might not be a problem here but I think it's better to keep the best practice if we believe it's not a lot of work. If it is we can leave like this.

Reply:
Lets make it a json: `{ "hash": "abcdefg...", salt: "123456..." }`